aries-site/src/examples/templates/page-layouts/anatomy/PageMarginAnatomy.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Box, Grid, Stack, Text } from 'grommet';
+import PropTypes from 'prop-types';
+
+// values chosen to best visually represent the margin area in diagram
+const largeColumns = ['13px', 'auto', '13px'];
+const smallColumns = ['8px', 'auto', '8px'];
+
+const MarginText = ({ label }) => (
+  <Stack anchor="bottom">
+    <Text alignSelf="center">{label}</Text>
+    {/* values chosen to best visually represent horizontal line in diagram. */}
+    <Box width="12px" height="1px" background="black" />
+  </Stack>
+);
+
+const GridViewPort = ({ columns, label, marginLabel, width }) => (
+  <Box gap="small" width={width}>
+    <Box direction="row" justify="between">
+      <MarginText label={marginLabel} />
+      <MarginText label={marginLabel} />
+    </Box>
+    <Grid
+      columns={columns}
+      // values chosen to best visually represent the margin area in diagram
+      rows={['20px', 'small', '20px']}
+      areas={[
+        ['left-margin', 'top-gap', 'right-margin'],
+        ['left-margin', 'viewpoint', 'right-margin'],
+        ['left-margin', 'bottom-gap', 'right-margin'],
+      ]}
+      border={{ style: 'dashed' }}
+      // border radius needed to match figma had to use in style
+      style={{ borderRadius: '4px' }}
```

Comment:
I believe round="xsmall" will result in `3px`, could we do that instead if it's reasonably close?

Reply:
since I'm using an inline style it doesn't recognize our t-shirt sizing so I have to use `px`

Reply:
More wondering if we remove the inline style and do `round="xsmall"`?

Reply:
`Grid` doesn't have the prop round 😞

Reply:
Does something like this work?
```
<Box round="xsmall">
  <Grid ... />
</Box>
```

Reply:
I tried this locally and I wasn't able to achieve the same outcome I can play around though